### DISCUSSION TITLE
Always 65 pull requests

### DISCUSSION THREAD

**Discussion by floli** [magit/magit#4555] [Support] [answered]
Hello,
I have a GitHub Enterprise forge configured:
```
(push '("github.my-company.com" "github.my-company.com/api/v3" "github.my-company.com" forge-github-repository)
                  forge-alist)
```
The project has many (220) open PRs, but magit always displays 65 as the number of PRs. Is there a fixed number limiting the number of displayed PRs? Can this number be changed?

When using something like visit-pull-request (N v p) all PRs are tthere...

Thanks!

**Comment by tarsius** [comment, marked as answer]
Yes, `forge-topic-list-limit`.

(Forge only has 14 options, so it should be easy to check if an option to do a certain thing does or does not exist.)

**Comment by floli** [comment]
Thanks for the reply! My fault was to look for magit variables, not for forge.

The documentation says "All unread topics are always shown.". That does not seem to work with my forge, as most of the PRs are unread, but still the number is 65 when the value of `forge-topic-list-limit` is `(60 . 5)`.

**Reply by tarsius** [reply]
Yes that seems to be outdated information. I've added this to my todos.
